cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ClusterLoadParameters.java

```diff
@@ -28,7 +28,7 @@
  * <pre>
  * Get the cluster load
  *    GET /kafkacruisecontrol/load?start=[START_TIMESTAMP]&amp;end=[END_TIMESTAMP]&amp;time=[END_TIMESTAMP]&amp;allow_capacity_estimation=[true/false]
- *    &amp;json=[true/false]
+ *    &amp;json=[true/false];&capacity=[true/false]
```

Comment:
`;&`->`&amp;`

Also can we rename this parameter to `capacity_only`? I had a offline discussion with @efeg , we want to change the default behavior of `load` endpoint (i.e. when `capacity_only` = `false`) to return both broker capacity and utilization for different resources ( I think we can make that change to a follow-up patch). In that case, `capacity_only` is more accurate about the purpose of this parameter.

Reply:
Sounds good! I think `capacity_only` makes more sense as well

Reply:
I'll rename the other parameter items to reflect this change as well